### PR TITLE
[ADD] add module for delivery

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@lmignon @acsonefho @legalsylvain can you check my PR, thx

File: shopinvader_delivery_carrier/models/backend.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
```
Comment (acsonefho) [inline shopinvader_delivery_carrier/models/backend.py]: Should be Odoo :)

File: shopinvader_delivery_carrier/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models
```
Comment (acsonefho) [inline shopinvader_delivery_carrier/models/sale.py]: Same here

**Review by acsonefho — COMMENTED** [review]
@sebastienbeau Minor changes :)

**Pull request merged**